packages/ember/tests/acceptance/sentry-errors-test.js

```diff
@@ -0,0 +1,161 @@
+import { test, module } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { find, click, visit } from '@ember/test-helpers';
+import { run } from '@ember/runloop';
+import Ember from 'ember';
+import sinon from 'sinon';
```

Comment:
Can we make sure we use `jest` for everything? I know this might be an ember default, but I would like to prevent multiple testing libraries.

Reply:
It's not really just a default, it's pretty much the only testing library that is tightly integrated with Ember. I prefer jest over qunit in general, but for Ember it's not nearly as nice to setup with other frameworks (the only other real option being Mocha) as the other helpers etc. are missing. We'd have to do a fair amount of groundwork to get it working with Jest.

On the other hand though, the guides and documentation are great for Ember, and aside from a few more tests for performance / profiling I don't imagine we'll have many to maintain.

Reply:
Ok I guess best tool for the job kind of thing. Let's keep with the defaults then 👍